### PR TITLE
Dashboard/Feed updated implementation Task: https://bit.ly/2J7w9DH

### PR DESCRIPTION
**Pull request by WRIOLtd**
https://github.com/webRunes/WRIO-InternetOS/issues/1610#issuecomment-506736886

### CODE REVIEW

File: src/base/components/Dashboard.js [hunk 1]
```diff
@@ -44,42 +36,23 @@ class DashboardPage extends React.Component {
...
-                          )
-                        })}
-                      </ul>
-                    </div>
-
-                  </td>
-                  <td>{sensors[11].item.variableMeasured.value}</td>
-
+                  <td><div className="dashboard-sensor-id">{sensorProductData.productID}</div></td>
+                  <td><a href={`https://imec.wr.io/testbed/${sensorProductData.productID}/`}>{sensorProductData.name}</a></td>
```
Comment (AlexeyAnshakov) [inline src/base/components/Dashboard.js]: these urls must not be hardcoded, list of urls must be taken from the parent json-ld.

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]

File: src/base/components/Dashboard.js (hunk 1 above)
Comment (WRIOdev) [reply src/base/components/Dashboard.js]: ok

File: src/base/components/FeedList.js [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
-              return (
-                <p>
-                  {filterdData.dateCreated.slice(12)} &nbsp;{" "}
-              {filterdData.item.variableMeasured.name == 'last-measurement'? filterdData.item.variableMeasured.value.map(item => <span>{item}&deg;{"C"} </span>):filterdData.item.variableMeasured.value}{(filterdData.item.variableMeasured.unitText == 'celsius' && filterdData.item.variableMeasured.name != 'last-measurement')? <span>&deg;{"C"}</span> :null}
-                </p>
-              );
-            })}
+          <h1>{filterFeed[0].dateCreated.slice(12,20)}</h1>
+          <p>State: {filterFeed[0].item.variableMeasured.value}</p>
+          <p>Temperature: {filterFeed[1].item.variableMeasured.value} &deg;{"C"}</p>
```
Comment (AlexeyAnshakov) [inline src/base/components/FeedList.js:14]: All item names must be not hardcoded but taken from keys.

File: src/base/components/Dashboard.js (hunk 1 above)
Comment (WRIOLtd) [reply src/base/components/Dashboard.js]: updated

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]

**Pull request merged**